### ISSUE TITLE
[1.2 RC] TypeError: Cannot read property '0' of undefined when creating app on Windows

### ISSUE THREAD

**Issue by martijnwalraven** [meteor/meteor#5088]
It seems the code added in https://github.com/meteor/meteor/commit/45f3133b24a7ffb25293963afa6d4ea5e7a37cdd may be causing some trouble. Because `options.appDir` is undefined, [this line](https://github.com/meteor/meteor/blob/release-1.2/tools/cli/commands.js#L600) throws an exception. It seems to me this should be `files.pathBasename(appPathAsEntered)` instead, but I don't understand why this is only a problem on Windows. @stubailo, do you understand what is going on here?

```
> meteor create testapp --release 1.2-rc.8

C:\Users\IEUser\AppData\Local\.meteor\packages\meteor-tool\1.1.7-rc.8\mt-os.windows.x86_32\dev_bundle\lib\node_modules\fibers\future.js:278
                                                throw(ex);
                                                      ^
TypeError: Cannot read property '0' of undefined
    at toDosPath (C:\Users\IEUser\AppData\Local\.meteor\packages\meteor-tool\1.1.7-rc.8\mt-os.windows.x86_32\tools\static-assets\server\mini-files.js:32:8)
    at C:\Users\IEUser\AppData\Local\.meteor\packages\meteor-tool\1.1.7-rc.8\mt-os.windows.x86_32\tools\static-assets\server\mini-files.js:82:16
    at Array.map (native)
    at Function._.map._.collect (C:\Users\IEUser\AppData\Local\.meteor\packages\meteor-tool\1.1.7-rc.8\mt-os.windows.x86_32\dev_bundle\lib\node_modules\underscore\underscore.js:97:56)
    at Object.pathBasename (C:\Users\IEUser\AppData\Local\.meteor\packages\meteor-tool\1.1.7-rc.8\mt-os.windows.x86_32\tools\static-assets\server\mini-files.js:79:16)
    at Command.buildCommands.minArgs [as func] (C:\tools\cli\commands.js:600:21)
    at C:\tools\cli\main.js:1377:23
```

**Comment by martijnwalraven** [comment]
Also, but probably unrelated to this specific issue, there seems to be something weird going on with the filenames in the stacktrace. The last two entries show their file being located in `C:\tools\...` instead of `C:\Users\IEUser\AppData\Local\.meteor\packages\meteor-tool\1.1.7-rc.8\mt-os.windows.x86_32\tools\...`.

**Comment by leocavalcante** [comment]
The workaround for now is create apps at current dir:
`meteor create . --release METEOR@1.2-rc.12`

But it still not have support for Android apps on Windows.
Any plans?

**Comment by stubailo** [comment]
Oops, I have fixed this and it will come out in the next RC. Sorry I wasn't more careful!

**Comment by stubailo** [comment]
Should be released now in `1.2-rc.15`. Thanks for reporting!
